.redhat-ci.yml

```diff
@@ -11,6 +11,7 @@ container:
 
 packages:
   - libasan
+  - git
```

Comment:
(And actually, we should be able to drop the other two -- we should try to keep it empty if it's already in the image, so that we don't even have to `makecache`)